cfme/intelligence/reports/widgets/__init__.py

```diff
@@ -132,6 +136,41 @@ def create(self, widget_class, *args, **kwargs):
         view.flash.assert_no_error()
         return dashboard_widget
 
+    def import_widget(self, filepath, cancel=False):
+        """
+        Import yaml files containing widget data
+        Args:
+            filepath (str): Complete path to the file
+            cancel (bool): If true, widgets will not be imported
+        """
+        view = navigate_to(self, "ImportExport")
+
+        view.fill({"upload_file": filepath})
+        view.upload_button.click()
+        view.flash.assert_no_error()
+
+        import_view = self.create_view(ImportExportWidgetsCommitView, wait='5s')
+        import_view.table.check_all()
+
+        if cancel:
+            import_view.cancel_button.click()
+            view.flash.assert_message("Widget import cancelled")
+        else:
+            import_view.commit_button.click()
+            import_view.flash.assert_no_error()
+
+    def export_widget(self, *widgets):
+        """
+        Export custom widgets
+        Args:
+            widgets (str, list): a single widget name or a list of widget names
```

Comment:
I think a better way to describe this type of arg is `A single or many widget names`

Reply:
Its actually always a tuple, regardless of whether one or many strings are passed.